src/test/java/org/apache/maven/plugins/javadoc/JavadocUtilTest.java

```diff
@@ -675,4 +680,23 @@ private void stopSilently(Server server) {
             // ignored
         }
     }
+
+    public void testQuotedArgument() throws Exception {
+
+        String value = "      org.apache.uima.analysis_component:\n      org.apache.uima.analysis_engine\n";
+
+        String arg = JavadocUtil.quotedArgument(value);
+        assertThat(arg).isEqualTo("'org.apache.uima.analysis_component:org.apache.uima.analysis_engine'");
```

Comment:
assertEquals is simpler and more direct

Reply:
done via 74b03e61bbe839f1ab08f8ceb8594e1a6052a37f